DFC.App.DiscoverSkillsCareers.Services/Services/NotificationService.cs

```diff
@@ -0,0 +1,44 @@
+﻿using DFC.App.DiscoverSkillsCareers.Models.Assessment;
+using DFC.App.DiscoverSkillsCareers.Models.Common;
+using DFC.App.DiscoverSkillsCareers.Services.Contracts;
+using Notify.Interfaces;
+using System;
+using System.Collections.Generic;
+
+namespace DFC.App.DiscoverSkillsCareers.Services.Services
+{
+    public class NotificationService : INotificationService
+    {
+        private readonly INotificationClient notificationClient;
+        private readonly NotifyOptions notifyOptions;
+
+        public NotificationService(INotificationClient notificationClient, NotifyOptions notifyOptions)
+        {
+            this.notificationClient = notificationClient;
+            this.notifyOptions = notifyOptions;
+        }
+
+        public static Dictionary<string, dynamic> GetPersonalisation(string domain, string sessionId) => new Dictionary<string, dynamic>
```

Comment:
Looks like the data in this static dictionary will be set from the first transient `INotificationService` instance and then reused subsequently. Is that what we want?